editor/src/components/canvas/canvas-actions.spec.tsx

```diff
@@ -30,6 +30,7 @@ describe('createDragState', () => {
         </div>
       </div>
       `),
+      'dont-await-first-dom-report',
```

Comment:
```suggestion
      'dont-await-first-dom-report'
```
Minor change to trigger workflows